pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -2808,6 +2827,7 @@ public void receiveParameterStatus() throws IOException, SQLException {
             value), PSQLState.CONNECTION_FAILURE);
 
       }
+      return;
```

Comment:
Unrelated to this PR so should be separate. This is a good idea though as it syncs up this block with how the rest are handled.